client/src/components/FilterMenu.js

```diff
@@ -45,40 +47,40 @@ class FilterMenu extends Component {
         this.props.changeLocation(latLng);
       })
       .catch(error => console.error('Error', error));
-  }
+  };
 
   toggleCategories = () => {
     this.setState(prevState => ({
       isShowCategories: !prevState.isShowCategories,
     }));
-  }
+  };
 
   toggleIndustries = () => {
     this.setState(prevState => ({
       isShowIndustries: !prevState.isShowIndustries,
     }));
-  }
+  };
 
   toggleSalary = () => {
     this.setState(prevState => ({
       isShowSalary: !prevState.isShowSalary,
     }));
-  }
+  };
 
-  onChangeSalary = (event) => {
+  onChangeSalary = event => {
     const newSalary = parseInt(event.target.value, 10);
-    if ((typeof newSalary) === 'number') {
+    if (typeof newSalary === 'number') {
       this.props.changeSalary(newSalary);
     }
-  }
+  };
 
   // source: https://github.com/hibiken/react-places-autocomplete
   renderFunc = ({ getInputProps, getSuggestionItemProps, suggestions }) => (
     <div className="autocomplete-root">
       <input {...getInputProps()} />
       <div className="autocomplete-dropdown-container">
         {suggestions.map(suggestion => (
-          <div {...getSuggestionItemProps(suggestion)}>
+          <div {...getSuggestionItemProps(suggestion)} key={suggestion.id}>
```

Comment:
I think we should generally use a key when mapping over things, I think this works (tested briefly).